content/engineering/processes/pull-request-template.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Pull Request Template"
+date: 2021-09-01T17:07:09+02:00
+draft: false
+description: "This is the template for pull requests"
+weight: 5
+---
+
+Create the `.github/pull_request_template.md` file and add the following content:
+
+```
+## Context
+
+What is the context in which the following changes will be made?
+
+## Story
+
+[Jira Story Link]()
+
+## Work done
+
+What steps have you taken to fix the problem outlined in the context section?
+
+## Checklist
+
+- [ ] Tests - The code been covered by tests (70% minimum)
+- [ ] Implementation - The functionality meets the acceptance criteria of the ticket
+- [ ] Executed - The code has been executed and appears functional
+- [ ] Environments configured - Any configuration updates have been added to the relevant environments
+- [ ] Code documentation - Newly added code is sufficiently documented
+- [ ] Readme - Necessary readme updated added
+
+## Notes
+
+Is there any other prudent information you would want to add?
+```
```

Comment:
If we want to go with this, please update on our handheld repo

Reply:
I think we should stick with our current template